### ISSUE TITLE
Problem with Windows

### ISSUE THREAD

**Issue by thardes2** [soldair/node-qrcode#51]
I think this is not working on a Windows OS right? the "qrcode.js" has the following lines of code

```
var QRCodeLib = require(__dirname+'/lib/qrcode-draw')
, terminalRender = require(__dirname+'/lib/termialrender.js')
, Canvas = require('canvas')
, fs = require('fs');
```
The delimter '/' is not always working on Windows, as I get the following error:

> module.js:442 Uncaught Error: Cannot find module C:\...\Documents\development\myProject/lib/qrcode-draw

**Comment by soldair** [comment]
hmm
i believe the problem is we need to remove the __dirname bit altogether and replace it with ./
this lib is rather old =)

if you want to make a pr ill happily merge it =)

**Comment by soldair** [comment]
ok i just fixed it =)
published to npm as + qrcode@0.4.3
